Server/Pages/Shipment/RegisterShipment.razor

```diff
@@ -0,0 +1,31 @@
+@using Repositories.Entities
+@using Repositories.Interfaces
+
+@code {
+
+    [Inject]
+    public ISenderRepository SenderRepository { get; set; }
+    [Inject]
+    public IPackageRepository PackageRepository { get; set; }
+    [Inject]
+    public IPostMachineRepository PostMachineRepository { get; set; }
+    public Package Package { get; set; }
```

Comment:
I would prefer for objects to be camel case and only methods and classes to be pascal case.

Reply:
Yeah, in the upcoming sprint we will have to decide on conventions, (_repository for example can be used), but caution is needed with [Inject] could be it that only public properties can be injected.